paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/playground/model/PlaygroundCheckoutModel.kt

```diff
@@ -103,26 +103,88 @@ sealed class CheckoutCustomer(val value: String) {
 }
 
 @Serializable
-data class CheckoutRequest(
+class CheckoutRequest private constructor(
     @SerialName("initialization")
-    val initialization: String,
+    val initialization: String?,
     @SerialName("customer")
-    val customer: String,
+    val customer: String?,
     @SerialName("currency")
-    val currency: String,
+    val currency: String?,
     @SerialName("mode")
-    val mode: String,
+    val mode: String?,
     @SerialName("set_shipping_address")
-    val set_shipping_address: Boolean,
+    val set_shipping_address: Boolean?,
     @SerialName("automatic_payment_methods")
-    val automatic_payment_methods: Boolean,
+    val automatic_payment_methods: Boolean?,
     @SerialName("use_link")
-    val use_link: Boolean,
+    val use_link: Boolean?,
```

Comment:
While you’re at it:

```suggestion
    val useLink: Boolean?,
```

Plus the other occurrences here.

Reply:
Yeah, I think it makes sense.